modules/image-versions.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * configuration/enable-offline-mode.adoc
+// * release-notes/<version>-release-notes.adoc
+//
+:_mod-docs-content-type: REFERENCE
+[id="image-versions_{context}"]
+= Image versions
+
+You can manually pull, retag, and push {product-title} images to your registry. The current version includes the following images:
+
+.{product-title} images
+[%header,cols="1,1,2"]
+|===
+|Image |Description |Current version
+
+|Main
+|Includes Central, Sensor, Admission controller, and Compliance. Also includes `roxctl` for use in continuous integration (CI) systems.
```

Comment:
```suggestion
|Includes Central, Sensor, Admission controller, and Compliance images. Also includes `roxctl` for use in continuous integration (CI) systems.
```

Reply:
Thank you @snarayan-redhat, good pick. However, these are note images, I've updated it to say `components` instead.